components/draw/draw.service.ts

```diff
@@ -434,7 +434,7 @@ export class HsDrawService {
     const confirmed = await dialog.waitResult();
     if (confirmed == 'yes') {
       this.HsMapService.map.removeLayer(this.selectedLayer);
-      if (this.selectedLayer.get('synchronize') == true) {
+      if (this.selectedLayer.get('definition')?.format == 'hs.format.WFS') {
```

Comment:
You can also use HsLayerSynchronizerService.isLayerSynchronizable(layer) for this

Reply:
Id have to move it to the save map layman.service first, because of circular dependency. But I guess it might make a sense to be able to reuse the thin

Reply:
I was afraid about that. Lets postpone moving it for version 3?